### PR TITLE
Problem: Missing feature for json output in CLI

### LINKED ISSUE #66: Problem: missing feature for json output
how about adding json output
-o json
-o yaml
-o text <- default

### PR DESCRIPTION
**Pull request by devashishdxt**
Solution: Added option to choose json output in CLI. Fixes #66.

### CODE REVIEW

File: solo-machine/src/command.rs [hunk 1]
```diff
@@ -205,3 +245,8 @@ fn print_stream(
 
     Ok(())
 }
+
+pub(crate) fn print_json(color_choice: ColorChoice, json: impl Display) -> Result<()> {
+    let mut stdout = StandardStream::stdout(color_choice);
+    print_stream(&mut stdout, &ColorSpec::new(), json)
```
Comment (tomtau) [inline solo-machine/src/command.rs:251]: I guess the default Display implementation for json should use the pretty-printed one: https://docs.serde.rs/serde_json/fn.to_writer_pretty.html ?

**Review by tomtau — APPROVED** [review]
looks all right -- good for @leejw51crypto to check whether it's what he expected

**Review by leejw51crypto — APPROVED** [review]
lgtm

File: solo-machine/src/command.rs (hunk 1 above)
Comment (devashishdxt) [reply solo-machine/src/command.rs:251]: Ok. I'll change to pretty printed json.

Comment (devashishdxt) [reply solo-machine/src/command.rs]: Done

**Pull request merged**